sys/mips/mips/pmap.c

```diff
@@ -1727,6 +1745,22 @@ pmap_try_insert_pv_entry(pmap_t pmap, vm_page_t mpte, vm_offset_t va,
 
 /*
  * pmap_remove_pte: do the things to unmap a page in a process
+ *
+ * Returns true if this was the last pte in the pt (and possibly the last pt in
```

Comment:
Please consistently capitalise PTE/PT/PD/PDE (unless specifically referring to variables by name).

Reply:
Good call; that's what comes of spreading the work across two days, I suppose.  If you see any other inconsistencies still lingering, please advise.